airgun/entities/contenthost.py

```diff
@@ -55,12 +59,44 @@ def execute_package_action(self, entity_name, action_type, value):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def execute_module_stream_action(self, entity_name, action_type, module_name, stream_version):
+        """Execute remote module_stream action on a content
+
+        :param entity_name: content host name to remotely execute package
+            action on
+        :param action_type: remote action to execute. This is dict with containing 'Action'
+            and 'is_Customize' keys. Action key value can be one of
+            5: 'Enable', 'Disable', 'Install', 'Update', 'Remove', 'Reset'
```

Comment:
`'Action'` or `'Actions'`? `'is_Customize'` or `'is_customize'`?
Documentation in docstring is not consistent with code. And we really want documentation to be true.

Reply:
needs to be updated

Reply:
done !